src/docbuilder/rustwide_builder.rs

```diff
@@ -312,7 +309,6 @@ impl RustwideBuilder {
     }
 
     pub fn build_local_package(&mut self, path: &Path) -> Result<bool> {
-        self.update_toolchain()?;
```

Comment:
this will remove updating (and installing?) the toolchain when you run `cratesfyi build crate ...` locally. 

Is this intentional? 
( I'm not necessarily against it, only wanted to know more)

Reply:
Yes. My thoughts were that if you are managing builds manually rather than using the queue builder you should also probably be managing updating the toolchain manually. I could make the `build crate` command do the update itself before the build if you disagree, or add a `--update-toolchain` flag to it to make it controllable.